content/docs/1.4/scalers/cron.md

```diff
@@ -0,0 +1,51 @@
++++
+title = "Cron"
+layout = "scaler"
+availability = "v1.0+"
+maintainer = "Community"
+description = "Scale applications based on Cron Schedules."
```

Comment:
```suggestion
description = "Scale applications based on a cron schedule."
```

Reply:
`availability = "v1.0+"` that's not correct as well.